src/trigger.py

```diff
@@ -16,6 +16,20 @@
 import kernelci.config
 import kernelci.data
 from kernelci.cli import Args, Command, parse_opts
+import urllib
+
+DEFAULT_URL = 'http://172.17.0.1:8001'
+
+
+def get_node_by_commit_id(commit_id):
```

Comment:
I think we should be adding this to `kernelci.data.kernelci_api` rather than creating requests from scratch in this file.